pages/Configuring/Binds.md

```diff
@@ -175,10 +175,10 @@ with your mouse.
 
 _Available mouse binds_:
 
-| Name | Description |
-| -----|------------ |
-| movewindow   | moves the active window   |
-| resizewindow | resizes the active window |
+| Name | Description | Params |
+| -----|------------ |--------|
+| movewindow   | moves the active window   | none |
+| resizewindow | resizes the active window | 1 - resize and keep window aspect ratio, 2 - resize and ignore `keepaspectratio` window rule/prop, anything else for normal resize |
```

Comment:
might be worth noting what happens when we leave this empty (default behavior of obeying the rules)

Reply:
done I think